src/tpg_tests.c

```diff
@@ -1357,26 +1359,66 @@ static void test_case_purge_cbs(test_case_info_t *tc_info)
         if (l4_cb->l4cb_test_case_id != tc_id)
             return true;
 
-        purge_cnt++;
+        cnt++;
 
         test_sm_purge(l4_cb, tc_info);
         test_callbacks[tc_type][l4_proto].sess_purge(l4_cb);
         return true;
     }
 
-    purge_cnt += test_purge_list(tc_info, &tc_info->tci_state.tos_to_init_cbs);
-    purge_cnt += test_purge_list(tc_info, &tc_info->tci_state.tos_to_open_cbs);
-    purge_cnt += test_purge_list(tc_info, &tc_info->tci_state.tos_to_close_cbs);
-    purge_cnt += test_purge_list(tc_info, &tc_info->tci_state.tos_to_send_cbs);
-    purge_cnt += test_purge_list(tc_info, &tc_info->tci_state.tos_closed_cbs);
+    cnt = test_purge_list(tc_info, &tc_info->tci_state.tos_to_init_cbs);
```

Comment:
Is this useful information?

Reply:
Maybe... maybe not. It was for me. :-)
I added these purge counts and associated debug logs as I was focused on TCP connection/teardown and wanted to see the various lists TCBs lived in. This investigation is what resulted in me adding the code to (at least) send the TCP RSTs when the client test stops. I left the code here assuming someone might find it beneficial in the future as it offers more visibility.